xbanish.c

```diff
@@ -530,7 +531,7 @@ snoop_legacy(Window win)
 		Button2MotionMask | Button3MotionMask | Button4MotionMask |
 		Button5MotionMask | ButtonMotionMask;
 
-	if (XQueryTree(dpy, win, &root, &parent, &kids, &nkids) == FALSE) {
+	if (XQueryTree(dpy, win, &root, &parent, &kids, &nkids) == false) {
```

Comment:
XQueryTree manpage says it returns 0 on failure. No need to include <stdbool.h> for this.
```suggestion
	if (XQueryTree(dpy, win, &root, &parent, &kids, &nkids) == 0) {
```